### ISSUE TITLE
Common confusion with car brand

### ISSUE THREAD

**Issue by JoshRackers** [bennybp/planets#10]
Users often mistake the planet Saturn with the now-defunct car brand.  Information should be added to avoid this misconception.

**Comment by dgasmith** [comment]
Well, as long as there is a citation I'm fine with it.

**Comment by bannanc** [comment]
Is there the same problem with Mercury?

**Comment by JoshRackers** [comment]
Good point.  Apparently naming a car brand after a planet is a bad idea...
